types/test/create.ts

```diff
@@ -0,0 +1,22 @@
+import {User} from './models/User';
```

Comment:
This is the only test file which includes enhanced testing. Which means, that it tests also
- if type inference works 
  - for just the target model values
  - for included models
  - for included models where included models is an array
- if it throws in case of 
  - wrong property name in target model values
  - right property name but wrong type
  - wrong property name in included models
  - wrong property name in included models, where included models is an array.

Since all other functions like `findOrBuild`, `bulkBuild` and so on are using the same type, which is `ModelValues <M>` , they mostly testing if type inference work and if it throws an error if anything is wrong.